features/step_definitions/record_of_guidance_steps.rb

```diff
@@ -0,0 +1,8 @@
+When(/^appointment details are captured$/) do
+  @page = GuidancePage.new
```

Comment:
Good point. My first thought was that we'd need access to it in the other step definitions but on looking at them again, the rest of them don't involve the guidance capture form